Bottom: 1055c3b0644dd96b0a94071820670dd82be75c5e
Top:    c6d9c709249a9d40f4ded6160fe0baf06ffec56b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 20:45:49 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
